tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -160,6 +161,40 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 
 			helper.CmdShouldPass("odo", "service", "delete", svcFullName, "-f")
 		})
+
+		It("should fail to create operand if a non existing param is provided", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs")
+			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(operators)
+			stdErr := helper.CmdShouldFail("odo", "service", "create", fmt.Sprintf("%s/EtcdCluster", etcdOperator), "-p", "unknown-param=4", "--project", commonVar.Project)
+			Expect(stdErr).To(ContainSubstring("the parameter unknown-param is not present in the Operand Schema"))
+		})
+
+		It("should be able to create a postgres database with parameters", func() {
+			// this project has the operator group installed hence we shouldn't delete it and
+			// use it for all namespace bound operators
+			projectName := "odo-operator-tests"
+			name := helper.RandString(6)
+			oc.SetProject(projectName)
```

Comment:
I'm not sure this will work with concurrent tests, as all will want to run on the same project

Reply:
wouldn't it work as we have overriden the `KUBECONFIG` for each test to be the `(context)/config`? and I think we are running tests in parallel, correct me if I am wrong @prietyc123 ?

Reply:
If it happens that 2 instances of this test are running at the same time, they will both create/delete resources on the same kubernetes namespace `odo-operator-tests`.

Reply:
I understand but not sure if a user in the test has the permissions to install an operator.

Reply:
Confirmed with @mohammedzee1000 that the user doesn't have permissions to install operators as part of the tests

Reply:
> If it happens that 2 instances of this test are running at the same time, they will both create/delete resources on the same kubernetes namespace `odo-operator-tests`.

also why would 2 instances of the same test even run?